test/conformance/certificate/http01/certificate.go

```diff
@@ -32,8 +32,8 @@ func TestHTTP01Challenge(t *testing.T) {
 	ctx, clients := context.Background(), test.Setup(t)
 
 	certDomains := [][]string{
-		{subDomain + ".example.com"},
-		{subDomain + "2.example.com", subDomain + "3.example.com"},
+		{subDomain + ".knative.dev"},
```

Comment:
@upodroid @kvmware @nak3 

Do you know if submitting HTTP01 requests to the subdomain of *.knative.dev exhausts some global quota that could affect knative.dev website from refreshing it's cert?

I'm assuming no because that could open up DoS attack - but asking just in case

Reply:
https://letsencrypt.org/docs/rate-limits/

I think these apply to both http01 and dns01 challenges

Reply:
thanks for the input - I updated the domain to something we don't use `knative-test.dev`